src/templates/tasks/detail.template.html

```diff
@@ -234,6 +234,20 @@ <h2>Task details{{IF [[task.getUsePreprocessor()]]}} (Preprocessor task){{ENDIF}
           {{ENDIF}}
         </td>
       </tr>
+      <tr>
+        <td>Use static chunking:</td>
+        <td>
+          {{IF [[task.getStaticChunks()]] == 0}}
```

Comment:
The values for the StaticChunks value are defined as constants in https://github.com/hashtopolis/server/blob/master/src/inc/defines/tasks.php#L40, so ideally these should be used (and in line 243 below as well). 
This can be used by using constants in the templating, e.g. `[[$DTaskStaticChunking::NORMAL]]`.